chcemvediet/apps/inforequests/admin.py

```diff
@@ -64,8 +88,16 @@ class InforequestAdmin(admin.ModelAdmin):
             u'applicant',
             ]
     inlines = [
+            BranchInline,
             ]
 
+    def get_formsets_with_inlines(self, request, obj=None):
+        for inline in self.get_inline_instances(request, obj):
+            # hide BranchInline in the add view
```

Comment:
@martinmacko47 Je tento komentar vhodny, alebo ho radsej vymazat?

Reply:
Radsej by som sa vyhol overridnutiu metody `get_formsets_with_inlines`. Idealne mat vsetky overridy nutne ku zobrazeniu inlinu iba na change obrazovke v ramci samotneho inlinu. Aby sa dali presunut do mixinu a nebolo treba zakazdym menit aj nieco v hlavnom admine.

Skus pogooglit, ci v `has_change_permission(self, request, obj=None)` v samotnom inline nevieme z `request` zistit, ci sme prave na change alebo add obrazovke. A podla toho vratit true alebo false.

Pozor pomocou `obj is None` v `has_change_permission` nevieme zistit, ci sme na change alebo add obrazovke, pretoze aj change obrazovka zavola `has_change_permission` aj bez `obj`. Neviem preco.

Podla [implementacie `get_inline_instances`](https://docs.djangoproject.com/en/1.8/_modules/django/contrib/admin/options/#ModelAdmin.get_inline_instances) sa inline zobrazi ak aspon jedno z `inline.has_add_permission`, `inline.has_change_permission` alebo `inline.has_delete_permission` vrati true. Cize ak upravime `has_change_permission` aby vracalo true iba na change obrazovke, tak potom sa inline zobrazi iba na change obrazovke. Je to sice hack, ale neviem ako to inac spravit elegantne.

Reply:
:ok:

Reply:
Pouzil som `request.resolver_match.url_name`, ktoreho hodnoty su:
``` 
u'inforequest_branch_change'
u'inforequest_branch_add'
u'inforequest_action_change`
...

```

Reply:
To zneje rozumne